compressor/base.py

```diff
@@ -36,17 +36,18 @@ class Compressor(object):
     Base compressor object to be subclassed for content type
     depending implementations details.
     """
-    type = None
 
-    def __init__(self, content=None, output_prefix=None, context=None, *args, **kwargs):
+    def __init__(self, content=None, output_prefix=None, context=None, filters=None):
```

Comment:
I prefer leaving `*args` and (more importantly) `**kwargs`, removing them is a backwards-incompatible change for people inheriting from it.